repo/packages/C/crate/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "/crate/{{crate/cluster-name}}",
```

Comment:
the full id must be a variable... you could add to vars together such as `{{framework.id-prefix}}{{crate.cluster-name}}`